.env.geome

```diff
@@ -1,4 +1,4 @@
-ISB_PORT=127.0.0.1:10000
+ISB_PORT=0.0.0.1:10000
```

Comment:
All of the ISB instance configs now bind to 0.0.0.0 in order to allow prometheus to gather metrics from them if necessary.